pkgs/jni/tool/wrapper_generators/generate_c_extensions.dart

```diff
@@ -56,6 +56,11 @@ const wrapperGetterDecl = '''
 FFI_PLUGIN_EXPORT $wrapperName* GetGlobalEnv();
 ''';
 
```

Comment:
(I can only add comments on modified lines.)

(And unrelated to this PR.)

Can we add to the preamble in generated files which script was run to generate the file?
Searching verbatim in the repository for a line from the preamble also works but is a a bit too dective style.